website/src/hugo/static/_redirects

```diff
@@ -0,0 +1,2 @@
+/stable/*   /v0.18/:splat   200
+/latest/*   /v0.19/:splat   200
```

Comment:
I _think_ 200 is the right thing to do for SEO, so we have unchanging URIs and can start to accrue page rank.  But that's a dark art.  Advice welcome here.